vispy/testing/tests/test_networkx_layout.py

```diff
@@ -0,0 +1,43 @@
+#format pep8
```

Comment:
Would you mind adding the header to the top of this to match the other modules in vispy (at least most of them):

```
# -*- coding: utf-8 -*-
# Copyright (c) Vispy Development Team. All Rights Reserved.
# Distributed under the (new) BSD License. See LICENSE.txt for more info.
```